src/udev/udev-manager.c

```diff
@@ -1135,10 +1135,12 @@ Manager* manager_new(void) {
         return manager;
 }
 
-static int listen_fds(int *ret_ctrl, int *ret_netlink) {
```

Comment:
hmm, i think setting the socket name to "varlink" is a *good* thing, I'd continue to do that? why change that?

sd_listen_fds_with_names() actually looks for "varlink", and it supports both connection fds and listener fds, it checks via SO_ACCEPTCON which one it is and then adds it either as connection or as listener.

Hence I am not entirely sure why this whole patch is necessary?